api/src/pcapi/alembic/versions/20221205T110352_7c6a155d45c8_add_is_active_educational_institution.py

```diff
@@ -0,0 +1,20 @@
+"""add_is_active_educational_institution
```

Comment:
```suggestion
"""add isActive field in educational_institution table
```

pour être plus lisible dans les logs alembic